src/Resolver/InvoiceFileResolver.php

```diff
@@ -63,13 +63,12 @@ public function __construct(
     public function resolveInvoicePath(InvoiceInterface $invoice): string
     {
         if (null === $invoice->getPath() || (null !== $invoice->getPath() && 'prod' !== $this->environment)) {
-            $invoicePath = $this->invoiceFileGenerator->generateFile($invoice);
-
-            $invoice->setPath($invoicePath);
+            $invoiceFilename = $this->invoiceFileGenerator->generateFile($invoice);
+            $invoice->setPath($invoiceFilename);
```

Comment:
Add a blank line above another type of algorithm operation. We separate associations, calls and input/output actions with a blank line ;)